pkg/project/project.go

```diff
@@ -242,15 +242,14 @@ func generateProjectName() string {
 func FindFileOrDirectory(directory, filename string) (string, bool) {
 	i := 0
 	for i < maxRecursiveIteration {
-		if isRootPath(directory) {
-			return "", false
-		}
-
 		if fileExists(filepath.Join(directory, filename)) {
 			return filepath.Join(directory, filename), true
 		}
 
 		directory = filepath.Clean(filepath.Join(directory, ".."))
+		if isRootPath(directory) {
```

Comment:
We want this check back at the top, because we don't want to look for a `.wakatime-project` file in the root folder.